### PR TITLE
Change the letter next to player name to be their starting letter instead of "A"

### PR DESCRIPTION
**Pull request by LittleEndu**
This PR changes the parser to use the first board state to figure out what the starting letter of each player was, then assign that to each player. The starting letter is then shown it the player listing.

Before:

Before

After:

After

~~As a bonus, placeholder game is changed to be AIAIA~~

### CODE REVIEW

File: src/placeholder-game.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ const players: PlayerDetails[] = [
   socketID: ("xx_" + name) as SocketID,
   index: i as PlayerIndex,
   killedStep: null,
+  startingLetter: i % 2 === 0 ? "a" : "i",
```
Comment (glenjamin) [inline src/placeholder-game.ts:33]: 😂 👏

File: src/parser.ts [hunk 2]
```diff
@@ -230,6 +230,7 @@ export function newParser() {
     SyncNewBoardState({ squaresWithLetters, playerScores }) {
       if (startIndex !== undefined && game.you.name === "") {
         identifyPlayerOne(squaresWithLetters);
+        getStartingLetters(squaresWithLetters);
```
Comment (glenjamin) [inline src/parser.ts:233]: I think these two functions could be merged together into a function called something like `identifyPlayers`

**Review by glenjamin — COMMENTED** [review]
Nice work on this

File: src/parser.ts (hunk 2 above)
Comment (LittleEndu) [reply src/parser.ts:233]: Had a version like this at first, but `identifyPlayerOne` has two paths it takes. I wanted to keep the code DRY.

Comment (glenjamin) [reply src/parser.ts:233]: Oh right, my initial read made me think that there was more repetition this way around, but you're right - they're not as similar as I first thought

**Pull request merged**